Umbraco-Cloud/Upgrades/Migrating-To-New-Infrastructure/index.md

```diff
@@ -0,0 +1,96 @@
+
+### Updated DNS settings - you might need to update settings
+Your projects need to be updated to use the new Cloudflare DNS settings. 
+All CNAME records must point to __dns.umbraco.io__. If you require an A-Record this must point to `104.19.191.28` or `104.19.208.28`. See the [Manage Hostnames documentation](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Manage-Hostnames/) for detailed instructions.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/GKUzEu_bgg0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+You can see if you are using the new Cloudflare DNS settings or the “old” Let’s Encrypt under Project settings -> Hostnames:
+
+## Before the migration
+A project is only scheduled for migration when all the requirements are met. If outgoing IP addresses on Umbraco Cloud are being used, i.e. adding them to an allow list, note that these will change when the project is migrated to the new infrastructure. In this case, it is recommended to add the new outgoing IP addresses to the allow-list prior to the migration. See the [Using external services with Umbraco Cloud documentation](https://our.umbraco.com/Documentation/Umbraco-Cloud/Set-Up/External-Services/#using-external-services-with-umbraco-cloud) for the full list and more information.
+
+
+No other actions are required prior to migration.
+
+## During the migration
+We kindly ask users to hold off working on the project during the 4-hour migration window to ensure that deployments and other Umbraco Cloud services are not interrupted. 
+
+Please do not:
+* Interact with the backoffice on Dev/Stage/Live env.
+* Update schema in the backoffice on Dev/Stage/Live env.
+* Update project settings in the Cloud portal
+* Initiate deployments between environments in the Cloud portal
+* Interact with Git repositories(sync/push/pull) 
+
+Work on the project(s) can resume immediately after receiving confirmation of a successful migration. 
+
+## After the migration
+
+### New Umbraco Cloud project URL
+The project URL will change from `my-project-alias.s1.umbraco.io` to `my-project-alias.euwest01.umbraco.io`. It is only the project URL that changes. Custom hostnames will continue to work as baefore. 
+
+### New GIT repository URL
+There will be an improved repository URL you can switch to, but __it is not required as part of the migration__. The GIT repository will still be accessible from the same URL as before and any custom domains assigned to the project will continue to work as expected. 
+
+We’re moving away from the rather verbose GUID-based URLs that are used on Cloud currently to something resembling what you already know from i.e. Github.
+
+    https://my-project-alias.scm.s1.umbraco.io/<GUID>.git 
+
+will become 
+
+    https://scm.umbraco.io/euwest01/my-project-alias.git
```

Comment:
Is this supposed to work @sitereactor ? 
As an example, this is the Git url of a migrated site: https://mbe-test.scm.euwest01.umbraco.io/7e576a93-a7ec-4ee6-bdc3-9e354bc978ba.git
So I believe the only thing that changes is the "s1" now is "euwest01", or am I missing something?

Reply:
This new type of scm hostname hasn't been enabled yet.

Reply:
OK, can you please update the information to be correct?